Update definition of scope for service principal permissions.

paulkel opened the pull request:
Service principal only requires contributor role for resource group, not whole subscription.

azurecla commented:
Hi __@DavidDeSloovere__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

paulkel committed:
Update definition of scope for service principal permissions.

Service principal only requires contributor role for resource group, not whole subscription.
